$NetBSD: patch-CMakeLists.txt,v 1.4 2023/08/06 11:59:42 yhardy Exp $

Find pkgsrc lua with pkg-config.

Unconditionally install desktop files and make dependencies explicit.

--- CMakeLists.txt.orig	2023-07-29 21:28:24.000000000 +0000
+++ CMakeLists.txt
@@ -19,7 +19,7 @@ set(CMAKE_PROJECT_VERSION ${PROJECT_VERS
 set(PROJECT_CONTACT "core.xournalpp@github.com")
 
 ## CMAKE_Variables
-set(CMAKE_MODULE_PATH "${PROJECT_SOURCE_DIR}/cmake/find" "${PROJECT_SOURCE_DIR}/cmake/include")
+set(CMAKE_MODULE_PATH "${PKGSRC_CMAKE_MODS}" "${PROJECT_SOURCE_DIR}/cmake/find" "${PROJECT_SOURCE_DIR}/cmake/include")
 set(CMAKE_CXX_STANDARD 17)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 set(CMAKE_CXX_EXTENSIONS OFF)
@@ -325,7 +325,7 @@ install(DIRECTORY resources
         )
 
 # Install desktop shortcuts for Linux
-if (${CMAKE_SYSTEM_NAME} MATCHES "Linux")
+if (TRUE)
     message("Installing desktop files")
 
     # Install icons
